services/post_service.py

```
@@ -0,0 +1,59 @@
+from fastapi.params import Depends
+
+from repositories.post_repository import PostRepository
+from schemas.post import PostsCreateRequest, PostsUpdateRequest
+
+
+class PostService:
+    def __init__(self,post_repo:Annotated[PostRepository, Depends(PostRepository)]) -> None:
+        self.post_repo = post_repo
+
+    @staticmethod
+    def _verify_author(post:dict, current_user_id: str):
```

Ruff (0.14.11)

Missing return type annotation for staticmethod `_verify_author`

Add return type annotation: `None`

(ANN205)